### ISSUE TITLE
Actions is failing if the label is missing

### ISSUE THREAD

**Issue by cortinico** [gabrieldonadel/actions-apply-version-label#9]
Here a link to the failing run:
https://github.com/facebook/react-native/runs/4965707511?check_suite_focus=true

The action fails with:

```
(node:1477) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Error: HttpError: Not Found
Error: Label Version: 0.67.1 doesn't seem to exist
```

While instead it should just report a warning.

**Comment by gabrieldonadel** [comment]
@cortinico this has just been fixed by https://github.com/gabrieldonadel/actions-apply-version-label/pull/11 and should be included in the next release

https://github.com/gabrieldonadel/actions-apply-version-label/actions/runs/1756805059

**Comment by cortinico** [comment]
> @cortinico this has just been fixed by #11 and should be included in the next release

Great stuff 🎉 Thanks for the super quick follow up

**Comment by gabrieldonadel** [comment]
[v0.0.3](https://github.com/gabrieldonadel/actions-apply-version-label/releases/tag/v0.0.3) is now released and here is the PR bumping the version on react-native repo https://github.com/facebook/react-native/pull/32979
